TODO.rst

```diff
@@ -0,0 +1,26 @@
+Planned features
+================
+
+Skeleton TODO
+-------------
+
+ - Plugin class definitions
+ - Hooks?
+ - Logging in general
+ - Logging in tests
+ - sketch out kolibri command
+ - implement settings pattern
+ - python setup.py sdist bdist_wininst upload
+ - versioning
+ - changelog
+ - Circle repo
+
+Decision TODO
+-------------
+
+#. Django 1.8 or 1.9 (LTS or latest?)
+#. CircleCI, right?
+#. Should JS / BDD tests be separated out? How to run optionally?
```

Comment:
I think we can defer decisions on integration testing and JS testing until we *have* something to test.